cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -130,13 +130,6 @@ func (s *server) makeQueryOptions() *queryApp.QueryOptions {
 	}
 }
 
-func (s *server) Shutdown(ctx context.Context) error {
-	var errs []error
-	if s.server != nil {
-		errs = append(errs, s.server.Close())
-	}
-	if s.jtracer != nil {
```

Comment:
because you are still manually instantiating jtracer you need to keep closing it.

Reply:
Noted, maybe that is causing the error